app/models/student_fields.rb

```diff
@@ -0,0 +1,34 @@
+module StudentFields
```

Comment:
can you add the frozen_string_literal magic comment to this class? That way the values in the arrays are also frozen (and the memory objects can be reused)